### PR TITLE
Adjust osu!taiko hit size and strong scale to exactly match osu!(stable)

### PR DESCRIPTION
**Pull request by frenzibyte**
Resolves the first two points in https://github.com/ppy/osu/issues/17422.

Values were pretty much extracted via comparisons between screenshots taken on stable and lazer (classic):

</details>
<details>
<summary>regular hit size comparison</summary>

CleanShot 2022-04-02 at 23 34 28@2x

</details>

<details>
<summary>strong hit size comparison</summary>

CleanShot 2022-04-02 at 23 33 06@2x

</details>

| | objects |
|-|-|
| before | CleanShot 2022-04-02 at 23 37 20@2x |
| after | CleanShot 2022-04-02 at 23 40 12@2x |
| stable | CleanShot 2022-04-02 at 23 37 14@2x |

### CODE REVIEW

**Comment by bdach** [comment]
seems to match visually, i didn't check stable code because i couldn't be bothered to when it looks close enough visually. not sure about applying this universally across all skins (including the new default skin) but the changes are small enough wherein this is maybe fine to do.

**Review by bdach — DISMISSED** [review]

File: osu.Game.Rulesets.Taiko/Objects/TaikoStrongableHitObject.cs [hunk 1]
```diff
@@ -17,7 +17,7 @@ public abstract class TaikoStrongableHitObject : TaikoHitObject
         /// <summary>
         /// Scale multiplier for a strong drawable taiko hit object.
         /// </summary>
-        public const float STRONG_SCALE = 1.4f;
+        public const float STRONG_SCALE = 1.525f;
```
Comment (peppy) [inline osu.Game.Rulesets.Taiko/Objects/TaikoStrongableHitObject.cs:20]: If we want to make this perfectly matching:

```suggestion
        public const float STRONG_SCALE = 1 / 0.65f;
```

Haven't confirmed visually so please double-check.

File: osu.Game.Rulesets.Taiko/Objects/TaikoHitObject.cs [hunk 2]
```diff
@@ -14,7 +14,7 @@ public abstract class TaikoHitObject : HitObject
         /// <summary>
         /// Default size of a drawable taiko hit object.
         /// </summary>
-        public const float DEFAULT_SIZE = 0.45f;
+        public const float DEFAULT_SIZE = 0.475f;
```
Comment (peppy) [inline osu.Game.Rulesets.Taiko/Objects/TaikoHitObject.cs:17]: This is a harder value to confirm against stable. In lazer we look to be calculating this as a ratio of the playfield height, whereas in stable there's nothing like this.

I'll leave the numbers I found here in case it helps confirm this ratio:

- `hitcircle` sprite is 128px wide
- `taikohitcircle` sprite is 113px wide
- Big hits are `1.0` scale, normal hits are `0.65` (see change below).

Maybe fine to leave this variable as is with visual comparisons, but best to check how things look after applying the suggestion below.

**Review by peppy — CHANGES_REQUESTED** [review]

File: osu.Game.Rulesets.Taiko/Objects/TaikoHitObject.cs (hunk 2 above)
Comment (frenzibyte) [reply osu.Game.Rulesets.Taiko/Objects/TaikoHitObject.cs:17]: Tried looking for anything from these numbers, yet there still seems to be a hidden constant factor applied here.

I firstly took a screenshot on a 1024x768 resolution, wherein no scale looks to be applied at the playfield (by pasting a sprite of the playfield background to the screenshot and checking if it matches 1:1).

Afterwards I took both `taikobigcircle` and `taikobigcircleoverlay` and pasted them over the screenshot without any scale applied, then noticed they are slightly smaller than what's drawn (circle = 113px, overlay = 128px):



For roughly perfect match, the pasted sprites need to have a scale of ~114.2% applied (circle becomes ~129px, overlay becomes ~146.2px):



Interestingly, the legacy hit target piece is also affected by this issue, and the scale values in master (0.73x for `approachcircle`, and 0.7x for `taikobigcircle`) seem like what stable actually has, since they need to be scaled with roughly the same ratio as above to display properly (which has been accidentally achieved in https://github.com/ppy/osu/pull/17621, since both `0.83 / 0.73` and `0.8 / 0.7` are equal to ~1.14, ~114%)

To summarize, only the taiko hit objects and the hit target look to be affected (~114.2% scaled from raw textures), the input drum and the playfield background display as expected. Unsure if that'll help in figuring this out, or raise more questions instead.

Comment (frenzibyte) [reply osu.Game.Rulesets.Taiko/Objects/TaikoHitObject.cs:17]: That being said, excluding the deduced scale factor, the correct constants for lazer would have to be:
 - `DEFAULT_STRONG_SIZE = 128 (strong hit size) / 200 (default playfield height)`
 - `DEFAULT_SIZE = DEFAULT_STRONG_SIZE * 0.65 (normal hit scale)`

File: osu.Game.Rulesets.Taiko/Objects/TaikoStrongableHitObject.cs (hunk 1 above)
Comment (frenzibyte) [reply osu.Game.Rulesets.Taiko/Objects/TaikoStrongableHitObject.cs:20]: Matches better than the current ratio on quick visual comparisons.

**Review by peppy — APPROVED** [review]
